news/2024/2024-01-28-vocaloid-mapping-contest-2.md

```diff
@@ -0,0 +1,175 @@
+
+\**Screening will only occur in modes with a large number of entries.*
+
+### Prizes
+
+What would VOCALOID be without its mascots? For a contest featuring **iroha(sasaki)** songs sung by Kagamine *Rin* and judged by Natsume*Rin*, the first-place osu! winner will receive a *custom VMC Winner Kagamine Rin plushie* handcrafted by none other than [Noffy](https://osu.ppy.sh/users/1541323)!
+
+| Placing | Prize(s) |  |
+| :-: | :-- | :-- |
+| 1st place | Custom *VMC 2 Winner* VOCALOID plushie\*, 6 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points), *VMC 2 Winner* profile badge | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
+| 2nd place | 2 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |
+| 3rd place | 1 month of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |
+
+\**Prize only for participants of the osu! game mode, due to its one-of-a-kind nature.*
+
+This contest is created by and for vocal synth fans to promote music, artists, and mappers. **All mappers who participate will receive constructive feedback from experienced mappers in the community.** That's arguably a more valuable prize in its own right!
+
+### Songs
+
+As proven by this contest, great things can happen when we promote and support the artists who make osu! great.
+
+Below we have selected 5 of iroha(sakaki)'s incredible works to you to pick from that were appropriate for contest mapping, and we're confident that something will inspire you!
+
+#### iroha(sasaki) - The Cuckoo Bird's Beautiful Youth (5:18 / ??? BPM)
+
+**Vocaloid Featured**: VY1 & Kagamine Rin
+
+**Lyricist**: π
+
+[Download the beatmap template here!](LINK)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+#### iroha(sasaki) - Shirley!! (4:59 / ??? BPM)
+
+**Vocaloid Featured**: Kagamine Rin
+
+**Lyricist**: π
+
+[Download the beatmap template here!](LINK)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+#### iroha(sasaki) - Devolution (4:26 / ??? BPM)
+
+**Vocaloid Featured**: Kagamine Rin
+
+**Lyricist**: kuma(alfred)
+
+[Download the beatmap template here!](LINK)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+#### iroha(sasaki) - 14 (Miku ver.) (4:55 / ??? BPM)
+
+**Vocaloid Featured**: Hatsune Miku
+
+**Lyricist**: π
+
+[Download the beatmap template here!](LINK)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+#### iroha(sasaki) - sigh (4:08 / ??? BPM)
+
+**Vocaloid Featured**: Kagamine Rin 
+
+**Lyricist**: iroha(sasaki)
+
+[Download the beatmap template here!](LINK)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Rules
```

Comment:
not gonna lie i don't feel like the technical tidbits fit well compared to the rest of the article tbh, especially considering how long the news post already is

i feel like it would be \*much\* better if these two sections are to be removed altogether and be replaced by sth like this

```suggestion
For more information regarding the rules of the contest and how to submit your map, feel free to check out the [VMC 2 wiki](/wiki/Contests/VMC/2).
```

Reply:
not opposed to it if others think it important

Reply:
all contest news posts have rules in them, so i don't see why removing here is necessary